src/main/resources/alma/common/fields.xml

```diff
@@ -442,5 +442,28 @@
         </data>
       </entity>
     </entity>
+    <!-- E-Book and digital copy selections (inofficial) -->
+    <entity name="" sameEntity="true" reset="true">
+      <data source="962  .e" name="id">
+        <regexp match="^[^Z][^D][^B].*" />
+        <regexp match="(.*)" format="https://lobid.org/collections#${1}"/>
+      </data>
```

Comment:
@dr0i I was referring to this. At the moment I have two following regexp which in my opinion should be combined but the results are cut short, therefor I have two regexp. If you can combine them, feel free to change it.

Reply:
If you combine these like `"^[^Z][^D][^B](.*)"` you would always loose the first three characters.
I think two regexp are fine, they are also easy to understand. So +1.